orm/idgen_bucket.go

```diff
@@ -49,3 +49,15 @@ type IDGeneratorFunc func(db weave.KVStore, obj CloneableData) ([]byte, error)
 func (i IDGeneratorFunc) NextVal(db weave.KVStore, obj CloneableData) ([]byte, error) {
 	return i(db, obj)
 }
+
+// ValidateSequence returns an error if this is not an 8-byte
+// as expected for orm.IDGenBucket
+func ValidateSequence(id []byte) error {
+	if len(id) == 0 {
+		return errors.Wrap(errors.ErrEmpty, "sequence missing")
```

Comment:
nit: let's do either "missing sequence" or "sequence is missing"

Reply:
`missing sequence` fewer words 👍